chainset.go

```diff
@@ -100,3 +104,61 @@ func (cs chainSet) Start(ctx context.Context, testName string, additionalGenesis
 
 	return eg.Wait()
 }
+
+// TrackBlocks initializes database tables and polls for transactions to be saved in the database.
+// This method is a nop if dbFile is blank.
+// Expected to be called after Start.
+func (cs chainSet) TrackBlocks(ctx context.Context, testName, dbFile, gitSha string) error {
+	if len(dbFile) == 0 {
```

Comment:
Comparing string length to zero is valid and correct of course, but I find it less direct to read than `dbFile == ""`. Seeing the zero length comparison tricks me into thinking `dbFile` is a slice.

Reply:
Ha, a matter of preference as I prefer `len(dbFile) == 0` for strings and bytes.